From: Michael Olbrich <m.olbrich@pengutronix.de>
Subject: HACK: remove SOCK_CLOEXEC usage

SOCK_CLOEXEC is not supported by OSELAS.Toolchain-1.99.3

Not for upstream.

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>

---
 udev/udevd.c |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

Index: udev-162/udev/udevd.c
===================================================================
--- udev-162.orig/udev/udevd.c
+++ udev-162/udev/udevd.c
@@ -982,10 +982,11 @@ static int init_notify(const char *state
 		goto finish;
 	}
 
-	if ((fd = socket(AF_UNIX, SOCK_DGRAM|SOCK_CLOEXEC, 0)) < 0) {
+	if ((fd = socket(AF_UNIX, SOCK_DGRAM, 0)) < 0) {
 		r = -errno;
 		goto finish;
 	}
+	util_set_fd_cloexec(fd);
 
 	memset(&sockaddr, 0, sizeof(sockaddr));
 	sockaddr.sa.sa_family = AF_UNIX;
